payme.php

```diff
@@ -167,6 +167,27 @@ public function callback()
                 $this->respond($this->error_invalid_json());
             }
 
+			/** 
+				Fix support php-fpm
```

Comment:
Hello @Fayozjon,
Can you convert this comment to one-line or block comment, instead of doc comment, please?